core/deployment/src/main/java/io/quarkus/deployment/builditem/ExcludeDependencyBuildItem.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Build item that defines dependencies that should be indexed. This can be used when a dependency does not contain
+ * a marker file (e.g. META-INF/beans.xml).
+ */
+public final class ExcludeDependencyBuildItem extends MultiBuildItem {
+    private final String groupId;
+    private final String artifactId;
+    private final String classifier;
```

Comment:
Maybe we could use the `Optional<String> classifier` instead so that we don't need to wrap the classifier [here](https://github.com/quarkusio/quarkus/pull/15781/files#diff-7a26f56a0e4b32b9824349ccb6729d9ed7063b4dae0dd3b217349560033e33daR143).

Reply:
I agree, I cloned the existing class "IndexDependencyBuildItem" which doesn't use Optional, but if you agree I switch immediately.